src/sentry/static/sentry/app/favicon.tsx

```diff
@@ -0,0 +1,25 @@
+function changeFavicon(theme: 'dark' | 'light'): void {
+  const n = document.querySelector<HTMLLinkElement>('[rel="icon"][type="image/png"]');
+  if (!n) {
+    return;
+  }
+
+  const path = n.href.split('/sentry/')[0];
+  n.href = `${path}/sentry/images/${theme === 'dark' ? 'favicon-dark' : 'favicon'}.png`;
+}
+
+function prefersDark(): boolean {
+  return window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
+}
+
+const updateFavicon = () => changeFavicon(prefersDark() ? 'dark' : 'light');
+
+export function setupFavicon() {
+  // Set favicon to dark on load
+  if (prefersDark()) {
```

Comment:
here you check `prefersDark` twice.. here and inside the update favicon func.. wouldn't it be better if you pass this info as a parameter?

Reply:
changed it